_vector-search/tutorials/ai-search-flows/building-flows.md

```diff
@@ -0,0 +1,629 @@
+            }
+        }
+    }
+}
+```
+{% include copy.html %}
+
+---
+
+## Hybrid search 
+
+Hybrid search combines keyword and vector search. This example demonstrates configuring hybrid search.
+
+### ML resources
+
+Create and deploy an [Amazon Titan Text Embeddings model on Amazon Bedrock](https://github.com/opensearch-project/dashboards-flow-framework/blob/main/documentation/models.md#amazon-bedrock-titan-text-embedding).
+
+### Index
+
+Ensure the index settings include `index.knn: true` and your index contains a `knn_vector` field specified in the mappings as follows:
+
+```json
+{
+    "settings": {
+        "index": {
+            "knn": true
+        }
+    },
+    "mappings": {
+        "properties": {
+            "<embedding_field_name>": {
+                "type": "knn_vector",
+                "dimension": "<embedding_size>"
+            }
+        }
+    }
+}
+```
+{% include copy.html %}
+
+### Ingest pipeline
+
+Configure a single ML inference processor. Map your input text to the `inputText` model input field. Optionally, map the output `embedding` to a new document field.
+
+### Search pipeline
+
+Configure an ML inference search request processor and a normalization processor.
+
+**For the ML inference processor**, map the query field containing the input text to the `inputText` model input field. Optionally, map the output `embedding` to a new field. Override the query to contain a `hybrid` query. Ensure to specify the `embedding_field`, `text_field`, and `text_field_input`:
+
+```json
+{
+    "_source": {
+        "excludes": [
+            "<embedding_field>"
+        ]
+    },
+    "query": {
+        "hybrid": {
+            "queries": [
+                {
+                    "match": {
+                        "<text_field>": {
+                            "query": "<text_field_input>"
+                        }
+                    }
+                },
+                {
+                    "knn": {
+                        "<embedding_field>": {
+                            "vector": ${embedding},
+                            "k": 10
+                        }
+                    }
+                }
+            ]
+        }
+    }
+}
+```
+{% include copy.html %}
+
+**For the normalization processor**, configure weights for each subquery. For more information, see the [hybrid search normalization processor example]({{site.url}}{{site.baseurl}}/search-plugins/hybrid-search/#step-4-configure-a-search-pipeline).
+
+---
+
+## Basic RAG (document summarization)
+
+This example demonstrates configuring basic retrieval-augmented generation (RAG).
+
+Because the connector blueprint and model interface may evolve, the following example demonstrates a connector blueprint that simplifies interaction with the [Claude v1 messages API](https://docs.anthropic.com/en/api/getting-started#examples). This blueprint abstracts much of the complexity, exposing only a single `prompt` field as input to the model.  
```

Comment:
In the first sentence, the second phrase doesn't logically follow the first (the example doesn't demonstrate a connector blueprint because it and the model interface may evolve). Please separate into two sentences.